tf/src/tf_echo.cpp

```diff
@@ -110,15 +121,15 @@ int main(int argc, char ** argv)
       {
         tf::StampedTransform echo_transform;
         echoListener.tf.lookupTransform(source_frameid, target_frameid, ros::Time(), echo_transform);
-        std::cout.precision(3);
+        std::cout.precision(precision);
         std::cout.setf(std::ios::fixed,std::ios::floatfield);
         std::cout << "At time " << echo_transform.stamp_.toSec() << std::endl;
         double yaw, pitch, roll;
         echo_transform.getBasis().getRPY(roll, pitch, yaw);
         tf::Quaternion q = echo_transform.getRotation();
         tf::Vector3 v = echo_transform.getOrigin();
         std::cout << "- Translation: [" << v.getX() << ", " << v.getY() << ", " << v.getZ() << "]" << std::endl;
-        std::cout << "- Rotation: in Quaternion [" << q.getX() << ", " << q.getY() << ", " 
+        std::cout << "- Rotation: in Quaternion [" << q.getX() << ", " << q.getY() << ", "
```

Comment:
Please avoid unrelated whitespace changes. They make maintenance and change auditing harder.